cmd/ignite/cmd/hang.go

```diff
@@ -0,0 +1,40 @@
+package cmd
+
+import (
+	"fmt"
+	"github.com/spf13/cobra"
+	"github.com/weaveworks/ignite/pkg/providers"
+	"io"
+	"os"
+	"os/signal"
+	"sync"
+)
+
+func NewCmdHang(out io.Writer) *cobra.Command {
```

Comment:
remove this before merge?

Reply:
or, maybe better, rename it to `daemon` for now

Reply:
I'll rename it to `daemon`